Feat/flavia lopes/classe cliente

Flavslg opened the pull request:
Criação da Classe Cliente

Flavslg committed:
Create cliente.dart

feat: Criação da Classe Cliente

Flavslg committed:
feat: Criação da Classe Cliente

Criação da Classe Cliente

guifxb reviewed (changes requested):
Alterar double? para não ser nullable
E iniciazilá-lo como opicional e valor 0

Flavslg committed:
fix: atributo dinheiro

- Ajuste da variável dinheiro para não nullable;
- Correção do parametro opcional dinheiro dentro do construtor;

guifxb reviewed (approved):
Belezinha

guimiranda0 reviewed (approved):
Show de bola !!

gabrielmattano reviewed (approved):
Gostei de iniciar com valor 0 como padrão. Pode mergear.
